src/components/Incidents/IncidentsPage.jsx

```diff
@@ -36,19 +41,20 @@ const IncidentsPage = ({ customDataSource, namespace }) => {
   const title = t('Incidents');
 
   const incidentTypeFilter = (t) => ({
-    filter: () => console.log('here should be filter function'),
+    filter: (filter, incident) => filterIncident(incident, filter),
     filterGroupName: t('Incident type'),
     isMatch: () => console.log('here should be filter state'),
     items: [
       { id: 'long-standing', title: t('Long standing') },
       { id: 'informative', title: t('Informative') },
       { id: 'inactive', title: t('Inactive') },
     ],
-    type: 'name',
+    type: 'informative',
```

Comment:
I think this type should reflect the filter grouping in this case should it be "incident-status" or something similar?

Reply:
I will update the type